tardis/conftest.py

```diff
@@ -2,6 +2,9 @@
 
 
 def pytest_addoption(parser):
+    parser.addoption("--run-slow", action="store_true",
+                     help="include running slow tests during run")
+
```

Comment:
An option for detecting the command line flag `--run-slow` has to be added in **conftest.py.** The task of detecting it is silently handled by pytest behind the curtains. :smile: